django/db/migrations/operations/models.py

```diff
@@ -528,6 +532,9 @@ def describe(self):
     def migration_name_fragment(self):
         return 'alter_%s_%s' % (self.name_lower, self.option_name)
 
+    def can_be_ignored_to_reduce_with(self, operation):
+        return self.name_lower == operation.name_lower
+
```

Comment:
Couldn't this possibly cause issues if `operation` is not a `ModelOperation` and happens to have a matching `name_lower` (e.g. `FieldOperation`)

Pretty sure we'll want something along the lines of

```python
def can_reduce_through(self, operation, app_label):
    return (
        super().can_reduce_through(operation, app_label) or
        (
            isinstance(operation, AlterTogetherOptionOperation) and
            type(operation) is not type(self)
        )
    )

Reply:
You're probably right, thanks for the feedback 👍